desktop/frontend/src/mock/appDetailInfo.ts

```diff
@@ -0,0 +1,29 @@
+export type TAppDetail = {
+  // 当前选择的版本信息
+  currentVersionInfo: {
+    version: string;
+    sha: string;
+    supportProcessor: string;
+    memory: string;
+  };
+  // 所有版本信息
+  versions: string[];
+  // 安装指引
+  installGuide: string[];
+  // 当前版本readme.md文档内容
+  currentReadme: string;
+};
+const appDetailInfo: TAppDetail = {
+  currentVersionInfo: {
+    version: 'latest',
+    sha: 'ca82a6dff817ec66f44342007202690a93763949',
+    supportProcessor: 'ARM/X86_64',
+    memory: '148.69MB'
+  },
+  versions: ['latest', 'v.8.0', 'v.5.6'],
+  installGuide: ['sealos pull myseq:latest', 'sealos run mysql latest'],
+  // 当前版本readme.md文档内容
```

Comment:
Use English

Reply:
done

Reply:
delete all the zh in comments~

Reply:
finished all the comments zh I can find. still keep this in mind for all the files that I maintain